specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3556,9 +3556,87 @@
         "clientId"
       ]
     },
+    "FlexProfile": {
+      "type": "object",
+      "properties": {
+        "scaleProfile": {
+          "$ref": "#/definitions/ScaleProfile",
+          "description": "Specifications on how to scale the AgentPool."
+        }
+      },
+      "description": "Specifications on flex Agentpool."
+    },
+    "ScaleProfile": {
+      "type": "object",
+      "properties": {
+        "manual": {
```

Comment:
I like this structure, just wondering on naming. Do we prefer `manual` or something like `fixed` or `static`?

Reply:
manual is more like in comparison with the future auto ones we plan to add, e.g., autoprovision.